core/system/src/mpu/vmpu_freescale_k64_mem.c

```diff
@@ -57,10 +58,37 @@ typedef struct
     uint32_t count;
 } TBoxACL;
 
-uint32_t g_mem_acl_count;
+uint32_t g_mem_acl_count, g_mpu_slot;
 static TMemACL g_mem_acl[UVISOR_MAX_ACLS];
 static TBoxACL g_mem_box[UVISOR_MAX_BOXES];
 
+int vmpu_mem_push_page_acl(uint32_t start_addr, uint32_t end_addr)
+{
+    MPU_Region * region;
+
+    /* Check that start and end address are aligned to 32-byte. */
+    if (start_addr & 0x1F || end_addr & 0x1F) {
+        return -1;
+    }
+
+    /* Insert into MPU regions in RR fashion. */
+    region = (MPU_Region *) MPU->WORD[g_mpu_slot];
+    region->STARTADDR   = start_addr;
```

Comment:
Do not align the `=` signs as it becomes a pain to keep them aligned later on when we change this code :)